contracts/bonding/BondingManager.sol

```diff
@@ -1386,7 +1440,7 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
      * @param _newPosPrev Address of previous transcoder in pool if the delegate is already in or joins the pool
      * @param _newPosNext Address of next transcoder in pool if the delegate is already in or joins the pool
      */
-    function processRebond(
+    function _processRebond(
```

Comment:
Thoughts on omitting the underscore for now to minimize the diff given that we still have a lot of internal functions in this contract that do not follow that convention (the ones that do only do so to avoid a naming collision)?